### ISSUE TITLE
[devel] Error when starting download with Python 3.9

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5763]
When starting a download, I get the following error with Python 3.9 on my Mac (`devel` branch):

```
Traceback (most recent call last):
  File "/Users/martijndevos/Documents/tribler/src/tribler-gui/tribler_gui/dialogs/startdownloaddialog.py", line 199, in on_received_metainfo
    metainfo = json.loads(unhexlify(response['metainfo']), encoding='latin-1')
  File "/Users/martijndevos/Documents/tribler/src/tribler-core/tribler_core/utilities/json_util.py", line 112, in loads
    return json.loads(s, *args, **kwargs)
  File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 359, in loads
    return cls(**kw).decode(s)
TypeError: __init__() got an unexpected keyword argument 'encoding'
```

Seems that there is another `encoding` keyword argument that can be removed.

**Comment by ichorid** [comment]
https://github.com/asottile/pyupgrade to the rescue?

**Comment by da2x** [comment]
I only get this crash when adding torrents whose info dictionary contains an `encoding`. Other torrents seem to work fine.

Example torrent:
`https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-12-04/2020-12-02-raspios-buster-armhf-lite.zip.torrent`

Work around:
Download via magnet instead:
`magnet:?xt=urn:btih:7e33a0eb71021d53ab3653ec253dba39818959d3&dn=2020-12-02-raspios-buster-armhf-lite.zip`

**Comment by devos50** [comment]
Fixed by #5806 already
